meerkat_abacus/util/__init__.py

```diff
@@ -50,20 +52,21 @@ def is_child(parent, child, locations):
     return False
 
 
-def epi_week(date):
+def epi_week_for_date(date):
     """
     calculate epi week
 
```

Comment:
Could we include here or something else how we deal with the edge cases. Do we have week 0 or week 53 for example?